test/message/promise_fast_reject.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
I'm asking why we assert the result is `'One'` when `'Two'` can also be a legitimate result.